deploy/templates/nstemplatetiers/team/ns_dev.yaml

```diff
@@ -64,7 +64,7 @@ objects:
     limits:
     - type: "Container"
       default:
-        cpu: 150m
+        cpu: 500m
```

Comment:
Hopefully, it won't break our pipelines. 
Luckily the new pipelines don't create so many containers, so we should be fine

Reply:
ignore my previous comment - the 10cores will ensure that it will run fine :-)